src/renderer/containers/ReplayBrowser/ReplayFile.tsx

```diff
@@ -83,19 +108,34 @@ export const ReplayFile: React.FC<ReplayFileProps> = ({
               <ReplayActionButton
                 label="More options"
                 onClick={(e) => {
+                  e.stopPropagation();
```

Comment:
Instead of passing in `e.stopPropagation()` manually in each of the `onClick` props, why not just set it to stop propagation at the `ReplayActionButton` level.